aws/rds/rds.tf

```diff
@@ -40,10 +40,13 @@ resource "aws_rds_cluster" "forms" {
   storage_encrypted           = true
   allow_major_version_upgrade = true
   copy_tags_to_snapshot       = true
+  apply_immediately           = var.env == "development" ? true : false
 
   serverlessv2_scaling_configuration {
-    max_capacity = 8
-    min_capacity = 2
+    max_capacity = var.env == "development" ? 4 : 8
+    min_capacity = var.env == "development" ? 0 : 2
```

Comment:
I've hit weird problems in the past with an ACU below 1, but they were all around the database trying to perform automatic updates, so this may not be an issue for you.

Reply:
So far it's been working great.  It spins down and spins back up when a connection is attempted.  I'm not too worried about the automatic updates as the RDS gets recreated every week.  I doubt it would ever make it to a maintenance window.